### PR TITLE
Capitalize message content in the alert body

### PR DESCRIPTION
**Pull request by infotexture**
We currently have 2 variants for the initial capitalization of the message content in the alert body of multi-line alerts:

- {name}
  **S**hared a video _(in iOS/Desktop conversation list subtext & macOS desktop notifications)_
- {name}
  **s**hared a video _(in iOS push notifications)_

This PR capitalizes the alert body to align with the presentation elsewhere per https://github.com/wireapp/copywriting/issues/15.

### CODE REVIEW

File: Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift [hunk 1]
```diff
@@ -479,10 +465,6 @@ extension ZMLocalNotificationTests_Message {
     }
 
     func testThatItCreatesAudioNotificationsCorrectly() {
-        //    "push.notification.add.audio.group" = "%1$@ shared an audio message";
-        //    "push.notification.add.audio.group.noconversationname" = "%1$@ shared an audio message";
-        //    "push.notification.add.audio.oneonone" = "%1$@ shared an audio message in a conversation";
-
         XCTAssertEqual(bodyForAssetNote(.audio, conversation: oneOnOneConversation, sender: sender), "shared an audio message")
```
Comment (infotexture) [inline Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift]: Did I miss this one in the strings?

Looks like this should be capitalized too:

> Shared an audio message

— or is this part of a concatenated string that continues from another part?

Comment (infotexture) [reply Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift]: Pushed additional commit d0d36bd to capitalize this one too.

Comment (infotexture) [reply Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift]: @vytis Sorry I missed this the first time around… 😓

Comment (vytis) [reply Tests/Source/Notifications/PushNotifications/ZMLocalNotificationTests_Message.swift]: Not a problem! Let's wait until the tests finish running and see if all is 👌

**Comment by vytis** [comment]
All seems peachy!

**Pull request merged**